views/mc-answer.pug

```diff
@@ -1,11 +1,34 @@
-.form-group
+if answerForm
```

Comment:
The indentation seem too much XD

Reply:
It is a tab indentation, we will have to format all files ... maybe this weekend?